aemedge/scripts/scripts.js

```diff
@@ -270,6 +270,16 @@ export function decorateButtons(element) {
           a.className = 'button secondary';
           threeup.classList.add('button-container');
         }
+
+        // steam button
+        if (
+          up.childNodes.length === 1
+          && up.tagName === 'STRONG'
+          && twoup.childNodes.length === 1
+          && (twoup.tagName === 'P' || twoup.tagName === 'DIV')) {
+          a.className = 'button steam';
+          twoup.classList.add('button-container');
+        }
```

Comment:
We actually have this style already as the "secondary" button type, so I'm not sure this "steam" variant is needed.